tests/st/calicoctl/test_crud.py

```diff
@@ -470,7 +470,7 @@ def test_bgpconfig(self):
 
         # Attempt to delete the default resource by name (i.e. without using a resource version).
         rc = calicoctl("delete bgpconfig %s" % name(rev0))
-        rc.assert_error(DELETE_DEFAULT)
+        rc.assert_no_error()
```

Comment:
How would deleting the default bgpconfig succeed? I thought we don't let anyone delete it from calicoctl, did we change that?

Reply:
yeah, just checked and it's now possible to remove the default bgp resource. maybe @robbrockbank has the info.

Reply:
just talked to Rob, we have disabled it for now, but for some tests. I'm not really convinced that's a good reason to disable the check, but for alpha release, it's not necessary and we have a backlog item to rethink this, so I'm not going to block this